tests/integration/test_schema_protobuf.py

```diff
@@ -1326,30 +1330,42 @@ async def test_protobuf_update_ordering(registry_async_client: Client) -> None:
 """
 
 
-@pytest.mark.parametrize(
-    "registry_cluster, status",
-    [({"config": {}}, 404), ({"config": {"use_protobuf_formatter": True}}, 200)],
-    indirect=["registry_cluster"],
```

Comment:
Not sure how this test was able to assert the behaviour, not it is really doing what it should do. 
Test against 2 SR, 1 without protobuf formatter and the other with the formatter.